documentation/testing/Decisions/Testing-Pipeline-SetupOptions.md

```diff
@@ -0,0 +1,50 @@
+﻿# Background
+Our projects involve two main solutions - `EducationBenchmarking.Platform` and `EducationBenchmarking.Web`. Each solution has its unique set of tests: unit tests, integration tests, functional API tests for `Platform`, and Feature/E2E tests along with accessibility tests for `Web`. The focus of our exploration is on optimising the testing pipelines for the functional API tests for `Platform` and Feature/E2E tests and accessibility tests for `Web`.
+
+## Context and Problem Statement
+In the current state of our development lifecycle, we find ourselves at a stage where the absence of established testing pipelines demands our attention. To fortify the reliability and effectiveness of our projects, we recognize the essential need to implement robust testing structures. We need a solution that fulfills the following criteria.
+
+### Criteria
+1. **Simple Tests:** We want to keep the tests as simple as possible to make them easily manageable.
+2. **Cost-Effective:** The solution we implement should be mindful of resource allocation and ensure cost-effectiveness without compromising on the quality of our testing processes.
```

Comment:
Do you have any thoughts on what we should be aiming for cost-wise and what timeframes we should aim for regarding feedback?

Reply:
i would say if we could save something by not setting up additional resource and not affect the output then it is cost effective. Timeframe would be as quick as possible once the changes are made.  I hope it answer your question.